.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+Cargo.lock
```

Comment:
Cargo no longer ignore the `Cargo.lock` file on `cargo new --lib`, and recommends it be committed, so let's also do that.
```suggestion
```